gemini.lisp

```diff
@@ -77,7 +85,11 @@ response is fetched, then return the meta and the (decoded) body."
       (format ssl-stream "~a~c~c" req #\return #\newline)
       (force-output ssl-stream)
       (let ((resp (parse-response (read-until ssl-stream #\newline))))
-        (values resp (read-all-stream ssl-stream))))))
+        (values resp (if (and (= (first resp) 20)
+                              (second resp)
+                              (string= (subseq (second resp) 0 4) "text"))
```

Comment:
To be sure here the check should need for "text/" (with the slash.)

I can fix this after merging eventually :)

Reply:
Done with 28a0032d4b0828483fdddf5831f2ea498ecb822d!